### PR TITLE
homepage now shows campaings associated with current user

### PR DESCRIPTION
**Pull request by JonathanCarlton**
aswell as a list of all campaigns, later this will be only campaigns the player is a part of

### CODE REVIEW

File: world-builder/app/models/user.rb [hunk 1]
```diff
@@ -3,6 +3,5 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-    has_many :characters
```
Comment (ebackhaus) [inline world-builder/app/models/user.rb:6]: This should be changed to 'has_many :user_campaigns' and 'has_many :user_campaigns' in campaign.rb should be changed too

Comment (JonathanCarlton) [reply world-builder/app/models/user.rb:6]: sounds good ill change that

Comment (JonathanCarlton) [reply world-builder/app/models/user.rb:6]: added

**Comment by JonathanCarlton** [comment]
when you create new users, the campaign shown on the right side (campaigns where user is a GM), may display a campaign different to what the user's name is. This is because the campaigns have user ids hard coded in the seed file, but since users aren't created when the database is set up, the campaigns are assigned to each user once the associated user id has been created.

**Pull request merged**